### ISSUE TITLE
Dropping bin/* files for development in the gem?

### ISSUE THREAD

**Issue by junaruga** [ruby/power_assert#35]
I find `bin/console` and `bin/setup` files are included in the gem file. Are these the files for a development, right? What do you think about not including the files in the power_assert gem file?

```
$ gem fetch power_assert
Fetching power_assert-2.0.0.gem
Downloaded power_assert-2.0.0

$ gem unpack power_assert-2.0.0.gem

$ ls power_assert-2.0.0/bin/
console*  setup*
```

Seeing the current `power_assert.gemspec` excluding specific files by `Array#reject`, perhaps a way to include only necessary files might be a way not to include the `bin/*` files.
ex. https://github.com/puma/puma/blob/e870ab69f03e44a80c60df8b3ac1b42b1374f907/puma.gemspec#L16-L17

**Comment by k-tsj** [comment]
Fixed by 9c631447da653cd27c2da5cb6d23be40c12dea45. 

Thanks for reporting.

**Comment by junaruga** [comment]
Thanks for working on it!
